src/distribution/mod.rs

```diff
@@ -101,6 +101,7 @@ mod exponential;
 mod gamma;
 mod gaussian;
 mod uniform;
+mod triangular;
```

Comment:
Let’s keep this list with `mod` and the one below with `pub use` sorted in alphabetical order.